Andrew.ardito/nodesupportpolicy (#10825)
DataDog/documentation: content/en/tracing/setup_overview/compatibility_requirements/nodejs.md

For example, if version 5.0.0 of `dd-trace` is released on May 4, 2023, then the 4.x.x release line is supported on a maintenance mode basis until May 4, 2024. During this maintenance mode period, security and bug patches will be applied whenever possible.

If you have any questions or concerns about our support for a particular version of dd-trace-js, please [reach out to support][3] to discuss.

### Node.js Version Support

When the Node.js project drops support for an LTS major release line (i.e. when it goes EOL), we will drop support in the next major version of dd-trace we release.
We will support that EOL version of Node.js for at least another year on a maintenance mode basis in the last major release line of dd-trace that supports it.

Some issues cannot be solved in dd-trace and instead must be solved in Node.js. When this happens and the Node.js release in question is EOL, it’s not possible to solve the issue.

> We will not make new releases of dd-trace providing specific support for Non-LTS Node.js major release lines (i.e. odd numbered versions)

Reviewer: ```suggestion
We do not make new releases of `dd-trace` to provide specific support for non-LTS Node.js major release lines (odd numbered versions).
```

> For the best level of support, we recommend always running the latest LTS release of Node.js, and the latest major version of dd-trace. Whatever release line of Node.js you use, we also encourage using the latest version of Node.js on that release line, to ensure you have the latest security fixes.

Reviewer: ```suggestion
For the best level of support, always run the latest LTS release of Node.js, and the latest major version of `dd-trace`. Whatever release line of Node.js you use, also use the latest version of Node.js on that release line, to ensure you have the latest security fixes.
```

> For more information about Node.js release, refer to the [official Node.js documentation][4].

Reviewer: ```suggestion
For more information about Node.js release, see the [official Node.js documentation][4].
```
